src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -615,7 +632,16 @@ export class RoslynLanguageServer {
             childProcess = cp.spawn(serverPath, args, cpOptions);
         }
 
-        return childProcess;
+        //Log any stderr/stdout messages in our output channel.
+        childProcess.stderr.on('data', (data: { toString: (arg0: any) => any }) =>
+            _channel.append('[stderr]' + (isString(data) ? data : data.toString(this._encoding)))
+        );
+        childProcess.stdout.on('data', (data: { toString: (arg0: any) => any }) =>
+            _channel.append('[stdout]' + (isString(data) ? data : data.toString(this._encoding)))
+        );
+
+        const protocol = await transport.onConnected();
```

Comment:
Should we log something to a trace to say we're waiting for the connection to come in? If anything goes wrong with the named pipe we might otherwise hang here with the process running but no connection.